Library/Formula/rethinkdb.rb

```diff
@@ -75,6 +73,11 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    shell_output("#{bin}/rethinkdb create -d test")
```

Comment:
If the maintainers think it's a trivial problem, we can move on. I already work around it using `shell_output`.